### PR TITLE
Add test for creating and clearing unpropagatable object

### PR DESCRIPTION
**Pull request by GinnyJI**
Convert test-issue-771.sh to issue_test.go

Tested: make e2e-test

### CODE REVIEW

File: incubator/hnc/test/e2e/issues_test.go [hunk 1]
```diff
@@ -209,4 +209,18 @@ var _ = Describe("Issues", func() {
 		// Expected: delete successfully
 		mustRun("kubectl delete subns", nsChild, "-n", nsParent)
 	})
+
+	It("Should create and delete an unpropagatable object - issue 771", func(){
```
Comment (yiqigao217) [inline incubator/hnc/test/e2e/issues_test.go:213]: Oh sorry that I missed this in the first review. It should say "Should **set and unset CannotPropagateObject condition** - issue 771".

File: incubator/hnc/test/e2e/issues_test.go [hunk 2]
```diff
@@ -209,4 +209,18 @@ var _ = Describe("Issues", func() {
...
+		// set up
+		mustRun("kubectl create ns", nsParent)
+		mustRun("kubectl create ns", nsChild)
+		mustRun("kubectl hns set", nsChild, "--parent", nsParent)
+		// Creating unpropagatable object; both namespaces should have a condition
+		mustRun("kubectl create rolebinding --clusterrole=cluster-admin --serviceaccount=default:default -n", nsParent, "foo")
+		// verify
+		runShouldContain("CannotPropagateObject", 1, "kubectl hns describe", nsParent)
```
Comment (adrianludwin) [inline incubator/hnc/test/e2e/issues_test.go:221]: The comment says "both namespaces should have a condition" but you only appear to be testing one namespace for the condition? Ditto on line 224, you only appear to test that one namespace does *not* have a condition.

Comment (yiqigao217) [reply incubator/hnc/test/e2e/issues_test.go:221]: Ah right, you should also add `runShouldContain` `CannotUpdateObject` condition in the child namespace.

File: incubator/hnc/test/e2e/issues_test.go (hunk 1 above)
Comment (GinnyJI) [reply incubator/hnc/test/e2e/issues_test.go]: done

File: incubator/hnc/test/e2e/issues_test.go (hunk 2 above)
Comment (GinnyJI) [reply incubator/hnc/test/e2e/issues_test.go:221]: added test for child

File: incubator/hnc/test/e2e/issues_test.go [hunk 3]
```diff
@@ -209,4 +209,20 @@ var _ = Describe("Issues", func() {
 		// Expected: delete successfully
 		mustRun("kubectl delete subns", nsChild, "-n", nsParent)
 	})
+
+	It("Should set and unset CannotPropagateObject condition - issue 771", func(){
```
Comment (yiqigao217) [inline incubator/hnc/test/e2e/issues_test.go:213]: one last nit "CannotPropagateObject/CannotUpdateObject" here

Comment (GinnyJI) [reply incubator/hnc/test/e2e/issues_test.go]: done

**Pull request merged**